Leanplum/Manager/LPRequestManager.h

```diff
@@ -0,0 +1,44 @@
+//
+//  LPRequestManager.h
+//  Leanplum
+//
+//  Created by Hrishikesh Amravatkar on 6/18/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "LPRequest.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LPRequestManager : NSObject
```

Comment:
can we call this `LPRequestStorageManager` or `LPRequestSQLiteStorageManager` `LPRequestSQLiteAdapter`?